src/main/java/UseCaseClasses/ScheduleEditor.java

```diff
@@ -1,5 +1,8 @@
-import events.EventFactory;
-import events.Event;
+package UseCaseClasses;
+
+import Entities.Person;
+import Entities.Schedule;
+import events.*;

```

Comment:
i don't think this is packaged correctly -- 'events' was a temporary measure, and those event subclasses should go under entities, shouldn't they?